src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -596,27 +572,16 @@ internal Type ArgumentType
         }
         #endregion
 
-        #region Public Members
-        public MemberInfo MemberInfo { get { return m_memberInfo; } }
-        public CustomAttributeTypedArgument TypedValue { get { return m_value; } }
-        public string MemberName { get { return MemberInfo.Name; } }
-        public bool IsField { get { return MemberInfo is FieldInfo; } }
-        #endregion
-
+        public MemberInfo MemberInfo => m_memberInfo;
+        public CustomAttributeTypedArgument TypedValue => m_value;
+        public string MemberName => MemberInfo.Name;
+        public bool IsField => MemberInfo is FieldInfo;
```

Comment:
IsField could be cached when setting type above, but infrequent (zero or one time) access

Reply:
Has 0 calls in coreclr, probably not worth it?